docs/usage.md

```diff
@@ -3,130 +3,317 @@
 
 # Usage
 
-This section aims to demonstrate how you can leverage FortiAI to generate playbooks, create response plans, understand Jinja, and ask questions among other things. It also provides steps on how to train the solution using your playbook collection to get results that reflect your organization's context and includes prompting tips to help you receive better and more targeted responses from FortiAI.
-
-FortiAI operates in two distinct modes &ndash; Playbook Blocks and Conversation Mode &ndash; to provide intelligent assistance and guidance to security professionals: 
-
-- The **Playbook Blocks** mode is active in the playbook designer and provides users with two options
-    - Playbook Generation
-    - Response Plans / Jinja / Playbook How To's
-
-- The Conversation mode is active in the list view and detail view of modules in FortiSOAR and provides users with the **Ask a question** option. 
-
-<table>
-    <tr>
-        <th>EXAMPLE</th>
-    </tr>
-    <tr>
-        <td>In the Conversation mode:
-            <ul>
-                <li>click the 'FortiAI' icon on the lower-right corner in the alert detail view.
-                    <p><img src="./res/ai_bot_icon.png" alt="Advisor bot icon in FortiSOAR"></p>
-                </li>
-                <li>Click the <strong>FortiAI</strong> to open the flyout to contextually communicate with the FortiAI
-                    <p><img src="./res/ai_bot_dialog.png" alt="Advisor dialog"></p>
-                </li>
-                <li>Click the <strong>Clear Conversation</strong> button to manually clear a conversation in FortiAI The <em>Clear Conversation</em> button clears the conversations only from the current Conversation mode and not from <strong>Playbook Generation Mode</strong>.</li>
-            </ul>
-        </td>
-    </tr>
-</table>
-
-<table>
-    <th>NOTE</th>
-    <td>When a user logs out, all conversations are automatically cleared.</td>
-</table>
-
-## Use cases
-
-### Build response plans using FortiAI
+The **FortiAI** solution pack offers a variety of functions through its **AI Assistant** to managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
 
-Consider a situation where you are assigned to create a plan of action for investigating a ransomware event.
+1. **Generate Playbook**: Generate, create a playbook or workflow, or playbook steps.
+2. **Navigate Pages**: Navigate, open, or move to a specific page or module.
+3. **Provide Context**: Load, retrieve, fetch, access, or provide more information about an *already opened* record.
+4. **Link Record**: Link or correlate a record with other relevant records. For example: correlating an indicator to an existing alert.
```

Comment:
Need to add this point (#4) in the release-notes

Reply:
Done